tock/utilization/views.py

```diff
@@ -58,29 +53,34 @@ def get_queryset(self):
 
     def get_context_data(self, **kwargs):
         context = super(GroupUtilizationView, self).get_context_data(**kwargs)
-        context.update({'org_totals': org_billing_context()}
-        )
+        context.update({'org_totals': org_billing_context()})
         return context
 
 
-class UtilizationAnalyticsView(LoginRequiredMixin, TemplateView):
-    template_name = "utilization/utilization_analytics.html"
-
-    def get_context_data(self, **kwargs):
-        context = super(UtilizationAnalyticsView, self).get_context_data(**kwargs)
+class FilteredAnalyticsView(LoginRequiredMixin):
```

Comment:
I don't see where this refactored class was used elsewhere. Was there a different version where this new super-class was useful? I don't have a problem with factoring this functionality out, but it doesn't seem related to the rest of this PR.

Reply:
Thanks @neilmb, that's definitely unrelated to the functionality and a remnant I didn't completely walk back.

My[ original approach](https://github.com/18F/tock/pull/1221/commits/ca2d2cfc8703789d7d3f8fa5fe8c4a847787b6da#diff-cdf622b15efe20a10cd3c1e46afc2d717720d093cbc6b11dda8028c24e0a3c24R165) had the project charts on another `utilization/` page and the associated view alongside this.  When I switched gears to handling it in the projects app I left `FilteredAnalyticsView`.

We'll leave it in but I should have called it out specifically or otherwise separated from the rest of these changes.